packages/react-native-bridge/android/publish-aztec-and-bridge.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# 1. Verify the version argument is passed. We use the same version for react-native-aztec and react-native-bridge libraries.
+VERSION=$1
+if [[ -z $VERSION ]]; then
+    echo "This script requires the Bintray version to be passed as an argument."
+    echo "Example usage: './publish-aztec-and-bridge.sh \$VERSION'"
+    exit 1
+fi
+
+# 2. Check if we can call gradlew successfully
+./gradlew --help > /dev/null 2>&1
+gradlew_exists=$?
+if [ $gradlew_exists -ne 0 ]; then
```

Comment:
😄 

I was wondering why there was no `set -e` at the start of the script. This useful error handling explains it 👍

Reply:
Could you clarify what you mean by that?

Reply:
When I started reading the script, I was surprised there was not `set -e` instruction, to "exit immediately if a simple command exits with a non-zero status" (see `man bash`).

I've seen `set -e` in most scripts I worked with so far, a useful way to ensure they don't continue running if one of the operations they perform fails.

Once I saw that this scripts handles the exit code of the commands it calls to give useful information to the caller, it became clear to me why there is no  `set -e` instruction.

Reply:
Oh, I see. I didn't know about `set -e`, thanks for pointing it out.

> Once I saw that this scripts handles the exit code of the commands it calls to give useful information to the caller, it became clear to me why there is no set -e instruction.

🙇